source/include/dqm4hep/StatusCodes.h

```diff
@@ -1,243 +0,0 @@
-/// \file StatusCodes.h
```

Comment:
Why did you removed this file ???

Reply:
I was trying to remove the file's changes from the pull request and used "git rm" which removed the whole file! I'm not sure how to undo it.